libs/cli/langgraph_cli/cli.py

```diff
@@ -425,37 +328,33 @@ def _build(
     """,
     required=True,
 )
-@click.option(
-    "--platform",
-    help="""Target platform(s) to build the docker image for.
-
-    \b
-    Example:
-        langgraph build --platform linux/amd64,linux/arm64
-    \b
-    """,
-)
 @click.option(
     "--base-image",
     hidden=True,
 )
-@cli.command(help="Build langgraph API server docker image")
+@click.argument("docker_build_args", nargs=-1, type=click.UNPROCESSED)
+@cli.command(
+    help="Build langgraph API server docker image",
+    context_settings=dict(
+        ignore_unknown_options=True,
+    ),
+)
 @log_command
 def build(
     config: pathlib.Path,
-    platform: Optional[str],
+    docker_build_args: Sequence[str],
     base_image: Optional[str],
     pull: bool,
     tag: str,
 ):
     with Runner() as runner, Progress(message="Pulling...") as set:
-        # check docker available
-        langgraph_cli.docker.check_capabilities(runner)
-        # open config
+        if shutil.which("docker") is None:
```

Comment:
ooc, why not use `check_capabilities`?

Reply:
Because that checks for docker compose etc, which more people don't have, and isn't needed to build images